### PR TITLE
Updated file arvore_binaria_de_busca.py

### PR DESCRIPTION
**Pull request by Ananyasingh2002**
I added the code below the Portuguese code section that I converted to English in Python.

### CODE REVIEW

**Comment by Ananyasingh2002** [comment]
Can I make more contributions to converting code to English?

File: src/python/arvore_binaria_de_busca.py [hunk 1]
```diff
@@ -182,3 +182,185 @@ def exclui(no, ch):
...
+
+
+
+
+
+
+
+
+"""
+Converted in English
```
Comment (kelvins) [inline src/python/arvore_binaria_de_busca.py:197]:
```suggestion
```

File: src/python/arvore_binaria_de_busca.py [hunk 2]
```diff
@@ -182,3 +182,185 @@ def exclui(no, ch):
...
+Binary Search Tree in Python in English
```
Comment (kelvins) [inline src/python/arvore_binaria_de_busca.py:198]:
```suggestion
Binary Search Tree
```

**Review by kelvins — CHANGES_REQUESTED** [review]
Hi @Ananyasingh2002, you should overwrite the existing code in pt-br with this new version in english, not append.

**Pull request closed without merging**